doc/rst/language/spec/tuples.rst

```diff
@@ -1184,7 +1201,7 @@ Predefined Functions and Methods on Tuples
 
 
 
-.. function:: proc isTuple(t: tuple) param
+.. function:: proc isTuple(type: t) param
```

Comment:
I am not sure we should have the `:` here, I think this is saying the formal name is `type` and the 'type' of `type` is `t`, but I think what you want is just `type t`, as that appears how it's written in `Types.chpl`

Reply:
Oops. Thanks for pointing that out!